src/coreclr/src/inc/loaderheap.h

```diff
@@ -439,17 +439,17 @@ class LoaderHeap : public UnlockedLoaderHeap, public ILoaderHeapBackout
     LoaderHeap(DWORD dwReserveBlockSize,
```

Comment:
>LoaderHeap [](start = 4, length = 10)

Why add a new parameter here instead of making the caller just allocate an UnlocekdLoaderHeap

Reply:
`UnlockedLoaderHeap` requires the heap to be created by the user, when `m_fExplicitControl == true` it doesn't reserve its own pages (https://github.com/dotnet/runtime/blob/master/src/coreclr/src/utilcode/loaderheap.cpp#L1093-L1096). Also `LoaderHeap` offers a backout scheme that I'm using to backout allocation of a short stub if it can't be used, it calls into the protected `UnlockedBackoutMem` but is only supported through `LoaderHeap`, which has tracker APIs for convenient backout.